distributed/comm/tests/test_ucx.py

```diff
@@ -156,6 +154,10 @@ def test_ucx_specific():
 
 @gen_test()
 def test_ucx_deserialize():
+    try:
+        from .test_comms import check_deserialize
```

Comment:
With this import at the top-level, I frequently run into this error (more details in the snippet that follows). @quasiben and I looked at this at one point and decided to comment it when doing some pair debugging. Would suggest we limit this `import` to this test where it is used and skip liberally if we can't get it to work. Though if someone knows a better way to solve it, please feel free to suggest 🙂

```python
distributed/utils.py:138: in _get_ip
    sock.connect((host, port))
E   OSError: [Errno 101] Network is unreachable
```

<details>

```python
==================================== ERRORS ====================================
_____________ ERROR collecting distributed/comm/tests/test_ucx.py ______________
distributed/utils.py:138: in _get_ip
    sock.connect((host, port))
E   OSError: [Errno 101] Network is unreachable

During handling of the above exception, another exception occurred:
distributed/comm/tests/test_ucx.py:15: in <module>
    from .test_comms import check_deserialize
<frozen importlib._bootstrap>:983: in _find_and_load
    ???
<frozen importlib._bootstrap>:967: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:677: in _load_unlocked
    ???
../../miniconda/envs/rapids13/lib/python3.7/site-packages/_pytest/assertion/rewrite.py:152: in exec_module
    exec(co, module.__dict__)
distributed/comm/tests/test_comms.py:48: in <module>
    EXTERNAL_IP6 = get_ipv6()
distributed/utils.py:169: in get_ipv6
    return _get_ip(host, port, family=socket.AF_INET6)
cytoolz/functoolz.pyx:476: in cytoolz.functoolz._memoize.__call__
    ???
distributed/utils.py:148: in _get_ip
    socket.gethostname(), port, family, socket.SOCK_DGRAM, socket.IPPROTO_UDP
../../miniconda/envs/rapids13/lib/python3.7/socket.py:752: in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
E   socket.gaierror: [Errno -5] No address associated with hostname
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 1.50s ===============================
```

</details>

Reply:
Should add this is being tracked in issue ( https://github.com/rapidsai/ucx-py/issues/461 ).